packages/schemas/src/date/helper.ts

```diff
@@ -273,6 +280,15 @@ export const getPlugin = ({
               },
             ],
           },
+          locale: {
```

Comment:
If my understanding is correct, this is a Locale used only for the UI, right?  
If so, it might be better to rename it to something like `localForUI`.